source/_components/sensor.ted5000.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "TED5000 electricity monitoring"
+description: "How to add a TED5000 to Home Assistant."
+date: 2016-09-27 11:19
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ted.png
+ha_category: Sensor
+ha_release: 0.29
```

Comment:
Wont be included in 0.29. No new features are included in the current release since it's been delayed already. I suggest you change this to 0.30.